### DISCUSSION TITLE
Navigation timeout on Windows 7 x64

### DISCUSSION THREAD

**Discussion by DerHOgi** [jef/streetmerchant#1060] [Help]
I tried to install streetmerchant on a Windows 7 x64 machine. No problems with the installation, but when i start the bot I only get navigation timeouts. If I set PAGE_TIMEOUT="0" nothing happens anymore.

Any ideas?

I also installed this on ubuntu with the same settings and everything is working fine so far. Unfortunately now I get 403 errors on two shops (saturn and mediamarkt). Looks like they blocked linux browsers or linux as operating system. I still can access the websites with Windows or macOS with the same ip. This is why I tried to get this to run on Windows 7.

**Comment by Vearol** [comment]
I managed to run it on raspberry pi, but getting timeouts as well.
There was one time, that it launched successfully, but only once :/

**Comment by DerHOgi** [comment]
No one has any ideas? I tried different things but without success.
If I manually open the browser at C:\streetmerchant\node_modules\puppeteer\.local-chromium\win64-818858\chrome-win\chrome.exe there are no problems. Browser-Window opens and loads every website I want. The bot only throws timeouts. Very strage.

**Comment by Vearol** [comment]
@DerHOgi , I got it running, but it is more like a random occasion. Just try running it over and over... eventually it should work. Meanwhile, I have an issue, that stock displayed incorrectly.

For the purpose of testing I have added some random item from amazon, which is in stock and available. The app reports it as out of stock.

**Comment by DerHOgi** [comment]
@Vearol But you use a raspberry pi, right? I have a ubuntu installation that works fine. I have only problems with the windows installation. Ubuntu is fine but some stores start to block ubuntu/linux with chrome (403 forbidden errors). In my case mediamarkt.ts and saturn.ts. Other stores still work.

**Comment by Vearol** [comment]
@DerHOgi , I wanted to hook up raspberry pi to a shitty tv in this rent apartment, but raspberry doesn't see it :)
So I run it on Ubuntu. With windows I suggest checking if it has anything to do with firewall.

**Comment by kewtz** [comment]
I have the same issue with rp4 using rasbian. 
If I set headless to "false" I can see chromium load and the websites load, a few navigation timeouts, but not as many as when headless is set to "true".

**Comment by ghost** [comment]
Slightly old thread I suppose, but I'm having the same issues. Running on raspbian (as a headless setup). I get the debug messages that lookups are starting/done, but don't get any other info messages. All I get are error messages saying that the Navigation timeout has been reached
